README.md

```diff
@@ -24,15 +24,16 @@ or rename a template from the templates dir
 cp node_modules/eslint-config-digitalbazaar/templates/node.js ./.eslintrc.js
 ```
 
-Instructions:
-- [ ] inside your .eslintrc.js add "eslint-config-digitalbazaar" to your extends array
-- [ ] then add envs node, browser, or mocha to the relevant dirs
+## Usage
+We export multiple config files that can be used to extend your rules.
+
+All rules can accessed via their shorthand: digitalbazaar.
 
 Example .eslintrc.js root setup:
 ```js
 module.exports = {
   root: true,
-  extends: ['eslint-config-digitalbazaar']
+  extends: ['eslint-config-digitalbazaar'] // you may use the shorthand digitalbazaar here
```

Comment:
I think we just need to pick a method, verbose or shorthand and stick with it.  If our convention is to do one or the other, there is no need to confuse developers about what they should be doing here.

Reply:
Is it true that there has already been agreement to use the verbose method?  It is my memory that there was at some point.

Reply:
there was, but 

1. the verbose method has now been implemented by most of the team. my thinking was if something went wrong they would not know which dir in node_modules was causing the error if you used the shorthand.

2. if we are using more composition then a developer will now have to write out the full route twice at least, shorthand seems preferable in that case.

Reply:
I've always been in favor of the shorthand.  But it wasn't worth fighting over.  +1 to using that if you all want.  If so, I'd just change the earlier text to something short saying rules can use or omit the `eslint-config-` prefix.  Rather than being verbose on the topic just link to the eslint docs.

Reply:
@mattcollier I have 4 PRs open currently so I changed the README to use the verbose path and while I do mention the shorthand I added a note that we recommend using the verbose path.